pkg/common/consumer/consumer_handler.go

```diff
@@ -131,7 +153,7 @@ func (consumer *SaramaConsumerHandler) ConsumeClaim(session sarama.ConsumerGroup
 		if mustMark {
 			session.MarkMessage(message, "") // Mark kafka message as processed
 			if ce := consumer.logger.Desugar().Check(zap.DebugLevel, "debugging"); ce != nil {
-				consumer.logger.Debugw("Message marked", zap.String("topic", message.Topic), zap.Binary("value", message.Value))
+				consumer.logger.Infow("Message marked", zap.String("topic", message.Topic), zap.Binary("value", message.Value))
```

Comment:
Do we really want to bubble that up as Info?  Isn't that to verbose?

Reply:
oh I didn't mean to change that. Thanks! fixing